SugarCRM Plugin - Incorrect Module Name

What type of report is this:

| Q  | A
| ---| ---
| Bug report? | X
| Feature request? | 
| Enhancement? | 

## Description:
When running the fetch leads command, the following error is reported in Mautic

Sugarcrm Error
`
Module Does Not Exist Contact get_entry_list GET
`

On the CRM side the error seen is
`
[ERROR] SoapHelperWebServices->checkSessionAndModuleAccess - module does not exists - Contact
[ERROR] End: SugarWebServiceImpl->get_entry_list - FAILED on checkSessionAndModuleAccess
[ERROR] 20: Module Does Not Exist<br>This module is not available on this server
`

Looking at the access logs it appears that the API does in fact refer to the module name Contact when it should be Contact**s**.
`
"GET /service/v4_1/rest.php?method=get_entry_list&input_type=JSON&response_type=JSON&rest_data=%7B%22session%22%3A%22jt6mtb1ect8q4l6ephish3qh13%22%2C%22module_name%22%3A%22Contact%22%2C%22query ...
`

## If a bug:

| Q   | A
| --- | ---
| Mautic version | 2.9.0
| PHP version | 7.0.18

### Steps to reproduce:
1. Setup integration with SuiteCRM 6 [Sugar Version 6.5.24 (Build 509)]
2. Run the  mautic:integration:fetchleads command
3. Leads and companies sync, contacts fail to sync

--- a/app/bundles/PluginBundle/Command/FetchLeadsCommand.php
+++ b/app/bundles/PluginBundle/Command/FetchLeadsCommand.php
@@ -152,7 +152,7 @@ class FetchLeadsCommand extends ContainerAwareCommand
                         $output->writeln('');
                         $output->writeln('<comment>'.$translator->trans('mautic.plugin.command.fetch.contacts.starting').'</comment>');
                         $contactList = [];
-                        $results     = $integrationObject->getLeads($params, null, $contactsExecuted, $contactList, 'Contact');
+                        $results     = $integrationObject->getLeads($params, null, $contactsExecuted, $contactList, $contactObjectName);
                         if (is_array($results)) {
                             list($justUpdated, $justCreated) = $results;
                             $updated += (int) $justUpdated;

